conda_build/skeletons/cpan.py

```diff
@@ -180,8 +177,24 @@ def __enter__(self):
             return dst
 
 
-def loose_version(ver):
-    return str(parse_version(str(ver)))
+def LooseVersion(version):
```

Comment:
Following PEP8 and snake case, that should be
```suggestion
def loose_version(version):
```

Reply:
I was trying to make it clear that we are wrapping the `LooseVersion` class creation by naming it this way, also that I was moving back to using `LooseVersion` instead of the `loose_version` function introduced in PR #2156.

If others agree, I'm fine doing the name change.  I'll do it offline and push a new commit since it will need to be changed throughout the file.

Reply:
Hmmm... this is a tiny bit of a gray area in my opinion.  I'd be inclined to keep LooseVersion, since the usage really matches the instantiation of a class anyway.  If changing it to PEP8/snake case, I think the function name should be more of a verb than a noun.  `get_loose_version` or `loosen_version` or something like that.

Reply:
I'll just keep LooseVersion in this PR then.  There are quite a few other improvements that I'm saving for future PR's to break things into smaller sets of changes.